apps/masterbots.ai/app/actions/subscriptions.ts

```
@@ -2,177 +2,177 @@
+		inputs = data.inputs.reduce(
+			(acc: any, input: { label: any }) => ({
+				...acc,
+				[input.label]: '',
+			}),
+			{},
+		)
+	} catch (error) {
+		console.error('Error fetching prompt details:', error)
+		error = (error as Error).message
+	} finally {
+		return { data, error, inputs }
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)